app/test_main.py

```diff
@@ -1,3 +1,13 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_zeros() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_should_convert_into_2_human_age() -> None:
+    assert get_human_age(24, 24) == [2, 2]
+
+
+def test_should_return_different_numbers() -> None:
+    assert get_human_age(29, 28) == [3, 2]
```

Comment:
Use `@pytest.mark.parametrize` to avoid code duplication and test all cases from the checklist including testing against raised errors